### PR TITLE
Add markdown to questions

### PR DESCRIPTION
**Pull request by ShawnCollinge**
Author: Shawn

## What changes were made?
Added markdown to question page and dropdown to view all the info from a question.

### CODE REVIEW

File: src/components/admin/TechnicalQuestionList.tsx
Comment (elimelt) [inline src/components/admin/TechnicalQuestionList.tsx:1]: Can we just reuse this elsewhere? i.e. does it need to be admin only?

File: src/types.ts [hunk 1]
```diff
@@ -125,8 +125,8 @@ export interface TechnicalQuestion {
   questionId: string;
   title: string;
   created: string; // ISO 8601 date string
-  createdBy: Member;
-  approvedBy: Member;
+  createdBy: string;
```
Comment (elimelt) [inline src/types.ts:128]: member id is a number iirc

File: src/types.ts [hunk 2]
```diff
@@ -140,8 +140,8 @@ export interface RawTechnicalQuestion {
   question_id: string;
   title: string;
   created: string;
-  created_by: Member;
-  approved_by: Member;
+  created_by: string;
+  approved_by: string;
```
Comment (elimelt) [inline src/types.ts:144]: same here

File: src/components/admin/TechnicalQuestionList.tsx
Comment (ShawnCollinge) [reply src/components/admin/TechnicalQuestionList.tsx:1]: Why would you want to display all questions outside of admin?

File: src/types.ts (hunk 1 above)
Comment (ShawnCollinge) [reply src/types.ts:128]: Yes but I'm returning it as a string of the actual name with the serializer.

File: src/types.ts (hunk 2 above)
Comment (ShawnCollinge) [reply src/types.ts:144]: Yes but I'm returning it as a string of the actual name with the serializer.

File: src/types.ts (hunk 1 above)
Comment (elimelt) [reply src/types.ts:128]: ohhhhhh I see

File: src/components/admin/TechnicalQuestionList.tsx
Comment (elimelt) [reply src/components/admin/TechnicalQuestionList.tsx:1]: factor out component for single question

Comment (ShawnCollinge) [reply src/components/admin/TechnicalQuestionList.tsx:1]: I guess you can create a new component for the internal part of the questions. We don't want a list or collapsible for single questions, or the edit button outside of admin IMO.

Comment (ShawnCollinge) [reply src/components/admin/TechnicalQuestionList.tsx:1]: The problem with that though is then you don't get the Topic or title. I'd probably just do something similar elsewhere idk if it makes sense to try to modify this code for it.

Comment (elimelt) [reply src/components/admin/TechnicalQuestionList.tsx:1]: nw I can just copy and paste

**Review by elimelt — APPROVED** [review]

**Review by elimelt — APPROVED** [review]

**Pull request merged**